Post component

zridi-mahmoud opened the pull request.

Commit:
feat: post component

Commit:
feat: post component storybooks

nerraou commented on frontend/src/components/atoms/feed/Post.tsx:
```diff
@@ -0,0 +1,100 @@
+  avatar?: string;
+}
+
+const UserCard = ({ user }: { user: User }) => {
```
Avoid inline type

oxidia replied:
Create
```ts
interface UserCardProps {
  user: User;
}

nerraou commented on frontend/src/components/atoms/feed/Post.tsx:
```diff
@@ -0,0 +1,100 @@
+}
+
+const LikeSection = ({ count, liked, onLike, postId }: LikeProps) => {
+  const formatCount = (count: number) => {
```
Avoid shadowing; check Eslent, preferably get it out of the component

nerraou commented on frontend/src/components/atoms/icons/outline/Like.tsx:
```diff
@@ -0,0 +1,26 @@
+        stroke="#FCF9E8"
+        stroke-width="2"
+        stroke-linecap="round"
+        stroke-linejoin="round"
```
Make the SVG props JSX compatible

nerraou reviewed (changes requested):
Post belongs to Molecules .kindly move it from Atoms

oxidia commented on frontend/src/components/atoms/feed/Post.tsx:
```diff
@@ -0,0 +1,100 @@
+  );
+};
+
+interface LikeProps {
```
rename to LikeSectionProps

zridi-mahmoud committed:
fix(post component): - avoid shadowing
- move the component to molecules
- add onlike action to the component's storybook

zridi-mahmoud committed:
fix(post component): - remove inline types
- fix typos
- fix icon's properties

oxidia committed:
Merge branch 'main' into post

oxidia committed:
chore(post): fix imports

nerraou reviewed (approved):
LGTM
